### DISCUSSION TITLE
Modal/modals: Persistent Feature/Prop

### DISCUSSION THREAD

**Discussion by imnash21** [mantinedev/mantine#6593] [Feature requests]
Persistent feature for Modal/modals

```
<Modal persistent ...>
    ...
</Modal>
```

**Comment by rtivital** [comment]
What does this prop do?

**Reply by imnash21** [reply]
To prevent closing the modal with animation like shaking or scaling up & down, modal can close only if user clicked the close button or the provided cancel button by user, for short the persistent are combination of closeOnClickOutside=false and closeOnEscape=false but with Animation that showing the user to like "Ohh you're accidentally clicked the Escape, Outside, or the Overlay"
